Sources/MapboxMaps/Foundation/BaseMapView.swift

```diff
@@ -297,7 +297,7 @@ extension BaseMapView: DelegatingMapClientDelegate {
         metalView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
         metalView.autoResizeDrawable = true
         metalView.contentScaleFactor = UIScreen.main.scale
-        metalView.contentMode = .center
+        metalView.contentMode = .scaleToFill
```

Comment:
@mapbox/maps-ios I've reviewed this in context of the updates to MapView/BaseMapView and am wondering if there is a reason that the contentMode is set to `.center` (default would be `.scaleToFill`). Thoughts? 

cc @alexshalamov

Reply:
Having reviewed this further, we do not want this view to scale as it would create a disconnect between the rendered content.